src/main/java/ch/njol/skript/Skript.java

```diff
@@ -683,7 +686,22 @@ protected void afterErrors() {
 					if (TestMode.DEV_MODE) {
 						runTests(); // Dev mode doesn't need a delay
 					} else {
-						Bukkit.getWorlds().get(0).getChunkAtAsync(100, 100).thenRun(() -> runTests());
+						World world = Bukkit.getWorlds().get(0);
+						if (isPaper())
+							world.getChunkAtAsync(100, 100).thenRun(() -> runTests());
+						else {
+							Bukkit.getServer().getPluginManager().registerEvents(new Listener() {
+								@EventHandler
+								public void onChunkLoad(ChunkLoadEvent event) {
+									Chunk chunk = event.getChunk();
+									if (chunk.getX() == 100 && chunk.getZ() == 100) {
+										runTests();
+										event.getHandlers().unregister(this);
+									}
+								}
+							}, Skript.getInstance());
+							world.getChunkAt(100, 100);
```

Comment:
getChunkAt loads the chunk synchronously so you can just make runTests run after this statement